### ISSUE TITLE
loss is high, can't converge?

### ISSUE THREAD

**Issue by w5688414** [yangjinming1062/SpeechRecognition_MASR#2]
> [1/1000][9294/15056]    Loss = 99.25045776367188
[1/1000][9295/15056]    Loss = 107.12692260742188
[1/1000][9296/15056]    Loss = 100.68484497070312
[1/1000][9297/15056]    Loss = 104.57418823242188
[1/1000][9298/15056]    Loss = 89.89228820800781
[1/1000][9299/15056]    Loss = 81.98886108398438
[1/1000][9300/15056]    Loss = 87.66160583496094
[1/1000][9301/15056]    Loss = 88.66828918457031
[1/1000][9302/15056]    Loss = 90.50955200195312
[1/1000][9303/15056]    Loss = 85.57133483886719
[1/1000][9304/15056]    Loss = 93.04979705810547
[1/1000][9305/15056]    Loss = 103.11563110351562

I found the loss is high, and the model can't converge

**Comment by yangjinming1062** [comment]
> > [1/1000][9294/15056]    Loss = 99.25045776367188
> > [1/1000][9295/15056]    Loss = 107.12692260742188
> > [1/1000][9296/15056]    Loss = 100.68484497070312
> > [1/1000][9297/15056]    Loss = 104.57418823242188
> > [1/1000][9298/15056]    Loss = 89.89228820800781
> > [1/1000][9299/15056]    Loss = 81.98886108398438
> > [1/1000][9300/15056]    Loss = 87.66160583496094
> > [1/1000][9301/15056]    Loss = 88.66828918457031
> > [1/1000][9302/15056]    Loss = 90.50955200195312
> > [1/1000][9303/15056]    Loss = 85.57133483886719
> > [1/1000][9304/15056]    Loss = 93.04979705810547
> > [1/1000][9305/15056]    Loss = 103.11563110351562
> 
> I found the loss is high, and the model can't converge

Please try reduce the learnrate ?

**Comment by w5688414** [comment]
I try to reduce the learning rate from 0.5 to 0.01, the loss is high so far:

> [1/1000][271/15056]     Loss = 88.97693634033203
[1/1000][272/15056]     Loss = 93.4842300415039
[1/1000][273/15056]     Loss = 91.98631286621094
[1/1000][274/15056]     Loss = 94.15042877197266
[1/1000][275/15056]     Loss = 93.7325668334961
[1/1000][276/15056]     Loss = 85.1528549194336
[1/1000][277/15056]     Loss = 88.96836853027344
[1/1000][278/15056]     Loss = 79.52326202392578
[1/1000][279/15056]     Loss = 83.9815902709961
[1/1000][280/15056]     Loss = 77.60595703125

**Comment by w5688414** [comment]
I change ctc loss to 

> loss = ctcloss(nn.functional.log_softmax(out,dim=2), y, out_lens, y_lens)
I want to know whether it is correct

> [1/1000][195/30111]     Loss = 9.160211563110352
torch.Size([504, 8, 6314])
[1/1000][196/30111]     Loss = 9.557342529296875
torch.Size([598, 8, 6314])
[1/1000][197/30111]     Loss = 9.782478332519531
torch.Size([614, 8, 6314])
[1/1000][198/30111]     Loss = 9.079712867736816
torch.Size([535, 8, 6314])
[1/1000][199/30111]     Loss = 8.902685165405273
torch.Size([460, 8, 6314])

**Comment by husin123** [comment]
hello,Do you know where can download the pretrained model?

**Comment by yangjinming1062** [comment]
> hello,Do you know where can download the pretrained model?

You can consult this URL:https://github.com/nobody132/masr

**Comment by husin123** [comment]
hello,This pre-training model has failed.Are there any other download links?



------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yangjinming1062/SpeechRecognition_MASR"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年8月13日(星期四) 上午10:10
收件人:&nbsp;"yangjinming1062/SpeechRecognition_MASR"<SpeechRecognition_MASR@noreply.github.com&gt;;
抄送:&nbsp;"1922821024"<1922821024@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [yangjinming1062/SpeechRecognition_MASR] loss is high, can't converge? (#2)





  
hello,Do you know where can download the pretrained model?
  
You can consult this URL:https://github.com/nobody132/masr

**Comment by husin123** [comment]
Hey, guys, can we add a WeChat to talk about speech recognition
?This is my wechat:ellison_202007

**Comment by husin123** [comment]
Hey, guys, can we add a WeChat to talk about speech recognition
?I also met the problem of loss too hight.
This is my wechat:ellison_202007
